packages/combo-box/test/lazy-loading.test.js

```diff
@@ -993,23 +993,6 @@ describe('lazy loading', () => {
           expect(comboBox.hasAttribute('focused')).to.be.false;
           expect(comboBox.value).to.equal('other value');
         });
-
-        it('should keep previous value if allow-custom-value is set', () => {
-          comboBox.allowCustomValue = true;
-          comboBox.open();
-          setInputValue(comboBox, 'other value');
-          comboBox.close();
-          expect(comboBox.value).to.eql('other value');
-
-          comboBox.focus();
-          // FIXME: fails when using `setInputValue()`
-          const filterValue = 'item 12';
-          comboBox.inputElement.value = filterValue;
-          comboBox.filter = filterValue;
-
-          expect(comboBox.value).to.eql('other value');
-          expect(comboBox.inputElement.value).to.eql('other value');
-        });
```

Comment:
This test started failing, but it overall doesn't seem to make sense. If you type a custom value while the data provider is loading and press <kbd>Enter</kbd>, it should get committed as a custom value whereas this test asserts that such a value should be reverted in that case to the previous custom one. So I dropped it.

Reply:
I agree that the test is rather confusing. Also it's already not working when we add `input` event, as mentioned in the comment. So I suppose it's ok to remove this test.